src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -184,7 +184,7 @@
             </trans-unit>
             <trans-unit id="49">
                 <source>The file was only partially uploaded.</source>
-                <target>Het bestand is niet geheel geüpload.</target>
```

Comment:
"Het bestand **is** slechts gedeeltelijk geüpload."

This is what gives the most results on Google.